docs/file-formats/kmx-file-format.md

```diff
@@ -197,29 +197,29 @@ If `dwSystemID` is non-zero, then it will be one of the following values:
 
 ## `COMP_GROUP`
 
-| ∆  | Bits | Name       | Description                                                                 |
-|----|------|------------|-----------------------------------------------------------------------------|
-| 0  |  32  | dpName     | offset to name of the group from keyboard source, `0x0` if not present      |
-| 4  |  32  | dpKeyArray | offset to array of `COMP_KEY` entries                                       |
-| 8  |  32  | dpMatch    | offset to string for the [`match`] statement output, `0x0` if not present   |
-| 12 |  32  | dpNoMatch  | offset to string for the [`nomatch`] statement output, `0x0` if not present |
-| 16 |  32  | cxKeyArray | number of `COMP_KEY` entries in `COMP_GROUP.dpKeyArray`                     |
-| 20 |  32  | fUsingKeys | `0x1` if group is `using keys`, `0x0` otherwise                             |
+| ∆  | Bits | Name         | Description                                                                 |
+|----|------|--------------|-----------------------------------------------------------------------------|
+| 0  |  32  | `dpName`     | offset to name of the group from keyboard source, `0x0` if not present      |
```

Comment:
> from keyboard source

So, the offset from the start of the file?

Reply:
Also... do we rely on this line for the `PostKeystroke` and `NewContext` groups?  Like... check during load if they have a matching name entry?

I see we _don't_ do this for the main / start group, but I suppose that the `begin ... > use` syntax for declaring a start group doesn't require a fixed start-group name.

Reply:
> Also... do we rely on this line for the `PostKeystroke` and `NewContext` groups? Like... check during load if they have a matching name entry?

No. See:

```
| 40    | `TSS_BEGIN_NEWCONTEXT`          | 0-1   | 15.0    | zero-based index of group referenced in [`begin NewContext`]
| 41    | `TSS_BEGIN_POSTKEYSTROKE`       | 0-1   | 15.0    | zero-based index of group referenced in [`begin PostKeystroke`]
```

Reply:
> So, the offset from the start of the file?

Yes, see Principles again.

Reply:
Oh... they're turned into system stores.  I'll admit I missed that.

Reply:
Those poor system stores. They have become a dumping ground for everything. For a brief moment I entertained the idea of making KMX+ a system store...